### ISSUE TITLE
False Positive: SA1000 reports on target-typed new expressions

### ISSUE THREAD

**Issue by Corniel** [DotNetAnalyzers/StyleCopAnalyzers#3350]
See: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/target-typed-new

``` C#
MyClass class = new();
```

Triggers a SA1000, claiming there should be a space after the new keyword.

**Comment by sharwell** [comment]
Duplicate of #3214
